### PR TITLE
pref: internationalize the avatar processing toolbar

### LINKED ISSUE #4296: 头像处理工具栏提示信息国际化
### 你当前使用的版本

v2.8.0-rc.1

### 描述一下此特性

如图所示，头像处理工具的提示信息均显示了英文，建议进行国际化处理。
image

### 附加信息

_No response_

### PR DESCRIPTION
**Pull request by LIlGG**
#### What type of PR is this?

/kind improvement

#### What this PR does / why we need it:

对头像处理组件的 toolbar 进行国际化处理。

#### Which issue(s) this PR fixes:

Fixes #4296 

#### Does this PR introduce a user-facing change?
```release-note
None
```

### CODE REVIEW

**Review by guqing — APPROVED** [review]
/approve
image

File: console/src/locales/en.yaml [hunk 1]
```diff
@@ -883,6 +883,13 @@ core:
           title: Crop Avatar
         remove:
           title: Are you sure you want to delete the avatar?
+        tooltips:
+          upload: Upload
+          zoom_in: ZoomIn
+          zoom_out: ZoomOut
+          flip_horizontal: FlipHorizontal
```
Comment (ruibaby) [inline console/src/locales/en.yaml:890]:
```suggestion
          flip_horizontal: Flip Horizontal
```

File: console/src/locales/en.yaml [hunk 2]
```diff
@@ -883,6 +883,13 @@ core:
...
+          flip_vertical: FlipVertical
```
Comment (ruibaby) [inline console/src/locales/en.yaml:891]:
```suggestion
          flip_vertical: Flip Vertical
```

**Review by ruibaby — CHANGES_REQUESTED** [review]

File: console/src/locales/en.yaml (hunk 2 above)
Comment (wan92hen) [reply console/src/locales/en.yaml:891]: Zoom In 和 Zoom Out 也需要改下

**Review by ruibaby — APPROVED** [review]
/lgtm

**Pull request merged**